src/test/java/com/kodilla/ecommercee/ProductRepositoryTestSuite.java

```diff
@@ -0,0 +1,135 @@
+package com.kodilla.ecommercee;
+
+import com.kodilla.ecommercee.domain.Group;
+import com.kodilla.ecommercee.domain.Product;
+import com.kodilla.ecommercee.repository.GroupRepository;
+import com.kodilla.ecommercee.repository.ProductRepository;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class ProductRepositoryTestSuite {
+
+    @Autowired
+    ProductRepository productRepository;
+
+    @Autowired
+    GroupRepository groupRepository;
+
+    Group group = new Group();
+    Product product = new Product();
+    Product product2 = new Product();
+
+    public void createData() {
+        group.setName("test group");
+        product.setName("test product");
+        product.setGroup(group);
+        product.setPrice(11);
+        group.getProducts().add(product);
+
+        product2.setName("test product nr 2");
+        product2.setGroup(group);
+        product2.setPrice(22);
+        group.getProducts().add(product2);
+    }
+
+    public void deleteData() {
+        try {
+            groupRepository.deleteById(group.getId());
```

Comment:
Aby testy były uniwersalne (może się zdarzyć że usunięcie grupy nie usunie produktów) to proponuję najpierw pousuwać oba produkty a potem grupę